socorro/lib/ooid.py

```diff
@@ -62,19 +62,19 @@ def dateAndDepthFromOoid(ooid):
     year = month = day = None
     try:
         day = int(ooid[-2:])
-    except:
+    except ValueError:
```

Comment:
For all of these, since we're not doing any checking on the `ooid` argument, it could be an empty string or `None` or something bogus. Prior to this change, those cases would get handled. After this change, they don't.

I'd add `TypeError`. We don't need `IndexError` since it's a substring. Are there other things we should handle?

Reply:
Ah, I didn't know int can throw a TypeError. I can't find a list of possible errors, but those two seem like a prudent pair.